frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetSubmariner/ClusterSetSubmariner.tsx

```diff
@@ -1,6 +1,6 @@
 /* Copyright Contributors to the Open Cluster Management project */
 
-import { ManagedClusterAddOn, ManagedClusterSetDefinition, ResourceErrorCode } from '../../../../../../resources'
+import { ManagedClusterAddOn, ManagedClusterSetDefinition, ResourceErrorCode, SubmarinerConfigDefinition } from '../../../../../../resources'
```

Comment:
The RBAC warning should be displayed if the user does not have read/write permissions to create submariner-broker (aka Broker object) in the ManagedClusterSet-broker namespace. 

Looking at this code, I feel the check is validating the `submarinerConfig` and not `submariner-broker` in `managedclusterset-broker` namespace.